### PR TITLE
Apple project completed

### PR DESCRIPTION
**Pull request by mkezfarias**
Hi, I just Finished my first solo project ( Apple clone ), submitting for review. thanks dagger

### CODE REVIEW

**Comment by ebukaume** [comment]
Hi @mkezmkez :wave: 

You have done a really nice job with this project :+1: . We just need to fix a few things and we would be good to go.


- [x] You commit and PR messages should be descriptive. 
_Note: In the future, always try to use more descriptive commit and PR messages. You can refer to [this guide](https://github.com/RomuloOliveira/commit-messages-guide) on how to get better with commit messages_
- [x]  The project has a descriptive README.md.
_Note: You need to always make your README to look professional because it is the "landing page" for your project. It is the first impression visitors to your repo get about you, so it is very important. You can refer to [this repo](https://github.com/othneildrew/Best-README-Template) to draw some inspiration_

 **Bottom**
* [x] It contains four elements **which are links** (to other articles or `#`)

**Kindly submit another code review request after making those changes**

Happy Coding :v: 
**Ebuka Umeokonkwo**
[Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Twitter](https://twitter.com/ebukaume)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                </div>
+            </main>
+        </div>
+
+        <aside class="relimg">
+            <div class="relcontainer">
+                <img alt="iPhone 5S" src="assets/promo_iphone5s.jpg">
+                <img alt="iPhone 5c" src="assets/promo_iphone5c.jpg">
+                <img alt="30 years of Mac" src="assets/promo_30_years.jpg">
+                <img alt="supplier responsibility" src="assets/promo_supplier_responsibility.jpg">
```
Comment (Redvanisation) [inline index.html]: These four elements should be wrapped in anchor tags `<a>`

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @mkezmkez  🌟

**Good work on this project, but we need to fix the issues below before moving on ⬇️** 

1. The 4 bottom elements should be wrapped in anchor tags `<a>`, check my comment below.

2. There is a big white spacing between these 4 bottom elements, please fix that and make them look like the original website.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

File: index.html (hunk 1 above)
Comment (mkezfarias) [reply index.html]: Done, will resubmit for review

**Review by uche-inyama — APPROVED** [review]
## Hi Mkezmkez!

Everything looks good now.

Go ahead and merge with the master branch :slightly_smiling_face: 

#### ```Have a great day``` :slightly_smiling_face:

**Pull request merged**